de.dlr.sc.virsat.cef.target/virsat_cef_integration.target

```diff
@@ -100,6 +65,42 @@
 		<unit id="org.eclipse.justj.openjdk.hotspot.jre.full.feature.group" version="0.0.0"/>
 		<repository location="https://download.eclipse.org/justj/jres/17/updates/release/17.0.11"/>
 		</location>
+			<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">

```

Comment:
Does this lead to problems if we set specific versions here? In the past we set the directory and versions to 0.0.0 to always pull the latest?

Reply:
Probably not, but I reset them to 0.0.0 anyway.